providers/base/tests/test_pipewire_utils.py

```diff
@@ -0,0 +1,634 @@
+                          "change-mask": [ "input-ports",\
+                                           "output-ports",\
+                                           "state", "props",\
+                                           "params" ],\
+                          "n-input-ports": 0,\
+                          "n-output-ports": 0,\
+                          "state": "suspended",\
+                          "error": null,\
+                          "props": {\
+                            "media.class": "Audio/Sink",\
+                            "factory.name": "support.node.driver",\
+                            "node.name": "Freewheel-Driver",\
+                            "priority.driver": 19000,\
+                            "node.group": "pipewire.freewheel",\
+                            "node.freewheel": true,\
+                            "factory.id": 10,\
+                            "clock.quantum-limit": 8192,\
+                            "node.driver": true,\
+                            "object.id": 29,\
+                            "object.serial": 29\
+                          },\
+                          "params": {\
+                          }\
+                        }\
+                      }]'
+
+    source_audio_node = '[{\
+                           "id": 30,\
+                           "type": "PipeWire:Interface:Node",\
+                           "version": 3,\
+                           "permissions": [ "r", "w", "x", "m" ],\
+                           "info": {\
+                             "max-input-ports": 0,\
+                             "max-output-ports": 0,\
+                             "change-mask": [ "input-ports",\
+                                              "output-ports",\
+                                              "state", "props",\
+                                              "params" ],\
+                             "n-input-ports": 0,\
+                             "n-output-ports": 0,\
+                             "state": "suspended",\
+                             "error": null,\
+                             "props": {\
+                               "media.class": "Audio/Source",\
+                               "factory.name": "support.node.driver",\
+                               "node.name": "Freewheel-Driver",\
+                               "priority.driver": 19000,\
+                               "node.group": "pipewire.freewheel",\
+                               "node.freewheel": true,\
+                               "factory.id": 10,\
+                               "clock.quantum-limit": 8192,\
+                               "node.driver": true,\
+                               "object.id": 29,\
+                               "object.serial": 29\
+                             },\
+                             "params": {\
+                             }\
+                           }\
+                         }]'
```

Comment:
please use multiline strings

Reply:
Change this to suggestion according to code review criteria, feel free to close it.